e2e/test/iothub/service/DevicesClientE2ETests.cs

```diff
@@ -201,48 +201,29 @@ public async Task DevicesClient_UpdateDevicesAsync_Works()
         {
             // arrange
 
-            var device1 = new Device(_idPrefix + Guid.NewGuid());
-            var device2 = new Device(_idPrefix + Guid.NewGuid());
-            var edge = new Device(_idPrefix + Guid.NewGuid());
-            using var serviceClient = new IotHubServiceClient(TestConfiguration.IotHub.ConnectionString);
+            IotHubServiceClient serviceClient = TestDevice.ServiceClient;
+            await using TestDevice testDevice1 = await TestDevice.GetTestDeviceAsync(_idPrefix).ConfigureAwait(false);
+            await using TestDevice testDevice2 = await TestDevice.GetTestDeviceAsync(_idPrefix).ConfigureAwait(false);
+            var newStatus = ClientStatus.Disabled;
+            testDevice1.Device.Status.Should().NotBe(newStatus);
+            testDevice2.Device.Status.Should().NotBe(newStatus);
 
-            try
-            {
-                Device addedDevice1 = await serviceClient.Devices.CreateAsync(device1).ConfigureAwait(false);
-                Device addedDevice2 = await serviceClient.Devices.CreateAsync(device2).ConfigureAwait(false);
-                Device addedEdge = await serviceClient.Devices.CreateAsync(edge).ConfigureAwait(false);
-
-                // act
+            // act
 
-                addedDevice1.Scope = addedEdge.Scope;
-                addedDevice2.Scope = addedEdge.Scope;
-                BulkRegistryOperationResult result = await serviceClient.Devices
-                    .SetAsync(new[] { addedDevice1, addedDevice2 })
-                    .ConfigureAwait(false);
+            testDevice1.Device.Status = testDevice2.Device.Status = newStatus;
+            BulkRegistryOperationResult result = await serviceClient.Devices
+                .SetAsync(new[] { testDevice1.Device, testDevice2.Device })
+                .ConfigureAwait(false);
 
-                // assert
+            // assert
 
-                result.IsSuccessful.Should().BeTrue();
+            result.IsSuccessful.Should().BeTrue();
 
-                Device actualDevice1 = await serviceClient.Devices.GetAsync(device1.Id).ConfigureAwait(false);
-                actualDevice1.Scope.Should().Be(addedEdge.Scope);
+            Device actualDevice1 = await serviceClient.Devices.GetAsync(testDevice1.Id).ConfigureAwait(false);
+            actualDevice1.Status.Should().Be(newStatus);
```

Comment:
Changing status is less work than updating scope (which requires having an Edge device created).